src/auth.ts

```diff
@@ -54,21 +82,34 @@ export function generate(
           <username>\${env.${escapeXML(username)}}</username>
           <password>\${env.${escapeXML(password)}}</password>
         </server>
+        <server>
```

Comment:
If a user doesn't specify a GPG key, won't this be empty?

Will that cause issues for users who don't want GPG authentication

Reply:
Users will see the following by default whether they want GPG authentication or not:
```
<settings>
    <servers>
      <server>
        <id>github</id>
        <username>${env.GITHUB_ACTOR}</username>
        <password>${env.GITHUB_TOKEN}</password>
      </server>
      <server>
        <id>gpg.passphrase</id>
        <passphrase>${env.GPG_PASSPHRASE}</passphrase>
      </server>
    </servers>
</settings>
```

And GPG_PASSPHRASE won't be provided so it will be empty string. This isn't a problem for users that don't want GPG -- maven will behave normally and will only grab information from this other server config if it needs it for the GPG plugin.